examples/decoding/plot_decoding_spatio_temporal_source.py

```diff
@@ -44,7 +49,7 @@
 
 # Setup for reading the raw data
 raw = io.read_raw_fif(raw_fname, preload=True)
-raw.filter(2, None, fir_design='firwin')  # replace baselining with high-pass
+raw.filter(0.1, None, fir_design='firwin')
```

Comment:
It probably doesn't do much, although it's possible our filter has better attenuation. Still worthwhile having it in the example if you consider it generally good practice, though.